descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "inventory",
-      "version": "13.3",
+      "version": "14.0",
```

Comment:
it looks like we should also increase version for "inventory-batch" interface

Reply:
@RuslanLavrov done, thank you

Reply:
thank you